### ISSUE TITLE
How to set Provisioning Profile in Build Settings (Step 2.5)

### ISSUE THREAD

**Issue by masarapmabuhay** [chrisamanse/LIKHA-Guide#14]
Salut! I'm now setting the Provisioning Profile that I downloaded as indicated in the LIKHA-Guide. I am now in Step 2.5 (App Store Deployment). The issue that I encountered is that I am unable to set the Provisioning Profile, because when I press the up/down arrows -> Other..., there is now way to select the file.

Also, for the code-signing identity, the only choice that is provided is my name as an iPhone Developer. How do I set it to be "Usbong Social Systems, Inc."? Perhaps, once I've set the Provisioning Profile, I'll be able to have more choices?

<img width="1022" alt="screen shot 2016-10-25 at 10 56 27 am" src="https://cloud.githubusercontent.com/assets/344066/19671499/bd8ab790-9aa1-11e6-9026-b6b3e11c5fa6.png">

Merci beaucoup!

**Comment by chrisamanse** [comment]
This always happens. After you installed the provisioning profile (opening the `*.mobileprovision` file), Xcode doesn't update the choices. It is usually resolved by restarting Xcode (Quit by going to Xcode -> Quit Xcode in the menu bar or `Cmd + Q`).

**Comment by masarapmabuhay** [comment]
Salut! Merci beaucoup pour ta réponse! I opened the `*.mobileprovision` file, and even without restarting Xcode, I was able to update the `Release` field of the `Provisioning Profile` section. 

However, for the `Code Signing Identity`, I am unable to update it to `iPhone Distribution: Usbong Social Systems, Inc.` It seems that only my name as iPhone Developer is available.

Merci encore une fois!
